### ISSUE TITLE
Remove Summary Warnings

### ISSUE THREAD

**Issue by dcamachoj** [jimschubert/query-tag-action#4]
When executing this action, there are a couple of warnings that appear in the Workflow Summary:

- Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/
- The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

The solutions are described in the provided links.
Thanks

**Comment by ApexModder** [comment]
Also running into these warnings when using this action

**Comment by hannesa2** [comment]
I see this as well

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/3314607/209436153-467b77f5-905f-4a38-9f53-6aace565902d.png">

**Comment by jimschubert** [comment]
These issues should now be resolved.

I moved `v2` to point to the new `v2.1` release. I was having trouble getting GitHub releases to show the updated commit hash. GitHub's actions runner pulled the correct hash for v2, but I briefly deleted and recreated the release so everything displayed correctly in both GitHub and the actions marketplace.

Tested each version in https://github.com/jimschubert/ossify

<img width="1418" alt="image" src="https://user-images.githubusercontent.com/109659/212723310-0f1b400f-f47d-4e73-9ec8-d0b3ac12a295.png">

I still need to update that project's docker/setup-buildx-action@v1, docker/login-action@v1 but all warnings are resolved for this action (see also screenshots in #5).

Shows v2 displaying the new output logging:
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/109659/212723667-37801337-1365-4afa-a040-fef442973b24.png">
